ulauncher/modes/extensions/extension_remote.py

```diff
@@ -198,16 +198,12 @@ def parse_extension_url(url: str) -> UrlParseResult:
 
     url_parts = urlparse(url)
     path = url_parts.path[1:]
+    host = url_parts.netloc
```

Comment:
`url_parts.netloc` will be an empty string if it's a local file, so we didn't have to handle this differently for the two cases.